components/utilities/dialog/index.jsx

```diff
@@ -294,12 +314,10 @@ const Dialog = createReactClass({
 	},
 
 	handleOpen () {
-		const scopedElement = this.dialogContent;
-
-		if (this.props.variant === 'popover' && scopedElement) {
+		if (this.props.variant === 'popover' && this.dialogContent) {
 			DOMElementFocus.storeActiveElement();
 			DOMElementFocus.setupScopedFocus({
-				ancestorElement: scopedElement.querySelector('.slds-popover'),
```

Comment:
Was kind of weird that we used to do a search for slds-popover, even though slds-popover is a class we control. Now since the Dialog no longer receives a div child that it's tightly bound to, and the Dialog is of a type of popover, we can just use the dialogContent reference directly. 

If we want to be a bit defensive, we can do the scan if the class isn't on dialogContent, but then there'd be a bug anyways and the tests should catch that. I don't think this is an appropriate time for defensive checks

Reply:
The focusing helpers were originally taken from jQuery UI, so DOM query snuck in.

Reply:
Still uses a dom reference, there's just no need to query, there's only one possible element it can be.